scripts/create-kms-key.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+alias="crypt4gh"
+
+if ! which jq > /dev/null; then
+    echo "This script needs jq. Please install and try again."
+    exit 1
+fi
+
+if ! which aws > /dev/null; then
+    echo "This script needs aws-cli. Please install and try again."
+    exit 1
+fi
+
+
+set -e
+
+key_arn=$(aws kms create-key | jq -r .KeyMetadata.Arn)
+aws kms create-alias --alias-name alias/${alias} --target-key-id $key_arn
```

Comment:
Isn't that a convenience script that you keep on your machine only?
Why do you want it in the repo?

Reply:
I can remove it, I just thought it would be useful for people to use if they wanted to store keys in kms.

Reply:
I have the feeling that people with kms credentials would know how to create that file already.
For people without aws knowledge, that would not help ;)